aesara/gradient.py

```diff
@@ -361,81 +357,89 @@ def _traverse(node):
         else:
             rval.append(seen_nodes[out.owner][out.owner.outputs.index(out)])
 
-    return format_as(using_list, using_tuple, rval)
+    using_list = isinstance(f, list)
+    using_tuple = isinstance(f, tuple)
+    return as_list_or_tuple(using_list, using_tuple, rval)
 
 
-def Lop(f, wrt, eval_points, consider_constant=None, disconnected_inputs="raise"):
-    """Computes the L operation on `f` with respect to `wrt` at `eval_points`.
+def Lop(
+    f: Union[Variable, Sequence[Variable]],
+    wrt: Union[Variable, Sequence[Variable]],
+    eval_points: Union[Variable, Sequence[Variable]],
+    consider_constant: Optional[Sequence[Variable]] = None,
+    disconnected_inputs: Literal["ignore", "warn", "raise"] = "raise",
+) -> Union[Optional[Variable], Sequence[Optional[Variable]]]:
+    """Computes the L-operator applied to `f` with respect to `wrt` at `eval_points`.
 
     Mathematically this stands for the Jacobian of `f` with respect to `wrt`
     left muliplied by the `eval_points`.
 
     Parameters
     ----------
-    f : :class:`~aesara.graph.basic.Variable` or list of Variables
-        `f` stands for the output of the computational graph to which you
-        want to apply the L operator
-    wrt : :class:`~aesara.graph.basic.Variable` or list of Variables
-        variables for which you compute the L operator of the expression
-        described by `f`
-    eval_points : :class:`~aesara.graph.basic.Variable` or list of Variables
-        evaluation points for each of the variables in `f`
+    f
+        The outputs of the computational graph to which the R-operator is
+        applied.
+    wrt
+        Variables for which the R-operator of `f` is computed.
+    eval_points
```

Comment:
All instances of `R-operator` should be `L-operator` here.

Reply:
Fixed in https://github.com/aesara-devs/aesara/pull/1087.